samples/client/petstore/lua/petstore/spec/category_spec.lua

```diff
@@ -0,0 +1,35 @@
+--[[
+Swagger Petstore
+
+This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.
+
+OpenAPI spec version: 1.0.0
+Contact: apiteam@swagger.io
+Generated by: https://github.com/swagger-api/swagger-codegen.git
+Swagger Codegen version: 2.4.0-SNAPSHOT
+
+]]
+
+local petstore_category = require "petstore.model.category"
```

Comment:
Usually I put this `require` inside of the `describe`. That way if you're only running some tests or mocking out something, it won't require it too early.